src/App.tsx

```diff
@@ -1,25 +1,65 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import './styles/App.css';
+import './styles/nav-top.css';
+import './styles/nav-middle.css';
+import './styles/nav-bottom.css';
+import './styles/bottom-bar.css';
+import './styles/filter.css';
+import Header from "./components/Header";
+import Nav from "./components/Nav";
+import Footer from "./components/Footer";
+import ShopPage from "./pages/shop/ShopPage";
+import CartPage from "./pages/cart/CartPage";
+import SingleBeerPage from "./pages/singlebeer/SingleBeerPage";
+import HomePage from './pages/home/HomePage';
+import ConnectionErrorPage from './pages/conectionError/ConnectionErrorPage';
+import ErrorPage from './pages/error/ErrorPage';
+import { Routes, Route } from "react-router-dom";
+import { useState } from "react";
+import getNumberOfProuductsInCart from './helpers/getNumberOfProuductsInCart';
+
 
 function App() {
+  
+  const [numberOfProductsInCart, setNumberOfProductsInCart] = useState(getNumberOfProuductsInCart())
+  const handleNumberOfProductsInCart = (numberOfProducts:number): void => {
```

Comment:
do we need this function?

Reply:
We do use "handleNumberOfProductsInCart" function for showing the number of the products of the cart on the cart icon in the header part of the page.
We invoke it from different pages.

Reply:
I mean we invoke it from different components.

Reply:
yes, but before you've updated the code, it was receiving an argument and then calling another function with the same argument. E.g. func = (arg) => anotherFunc(arg). So calling func(arg) is the same as calling anotherFunc(arg). There's no meaning in having the function func

Reply:
oki doki :)